### ISSUE TITLE
Multiple alert in the same page

### ISSUE THREAD

**Issue by Sabe67** [vuejs-pt/vue-alert#4]
Hello,

Can we have multiple "<vue-alert></vue-alert>" on the same page and show the good one when it's needed ?

Thank you for your help.

**Comment by davidpicarra** [comment]
Hi @Sabe67 
You can have multiple vue-alerts in the same page if you put each vue-alert in it's own component

**Comment by Sabe67** [comment]
Thank you for your answer.
How it works if I have multiple vue-alert in the same component ?

**Comment by davidpicarra** [comment]
You can only have one vue-alert in a component. Can you not re-use the same vue-alert to show different messages?

**Comment by Sabe67** [comment]
Yes I can, but it would be nice if we could choose an vue-alert with an ID or something like this.

**Comment by davidpicarra** [comment]
At the moment that's not possible, will consider this suggestion for future features!
